### ISSUE TITLE
\edtext spanning \pend\pstart -- no error message

### ISSUE THREAD

**Issue by floriandk** [maieul/ledmac#763]
# Version of reledmac / reledpar (if that is a development version, please precise the branch)
reledmac 2018/05/02 v2.26.9
reledpar 2018/05/02 v2.21.8

# If this is a newly introduced problem, please indicate the last working version of the package(s)
all the way back to TL 15

# Does the problem happen only when combining with some packages?
no

# Does the problem happens both in parallel typesetting and normal typesetting?
yes

# Description of the problem
`\edtext` should not be spanning over section-borders (`\pend`/`\pstart`). If it does anyway, an `Error: \pend must be used within a numbered section.` is reported.

But when the wrong `\edtext` has a `\lemma` attached there is no error-message. Still the wrong coding can have strange effects that are difficult to track.

(I've just spent *many* hours trying to debug code that compiled properly using only `reledmac` but broke off without obvious reason when set up with `reledpar`. In the end an `\edtext` spanning a section border at a completely different place was the mistake...)

 It would be very useful, if the error message would come up, also when lemma is used.

# Minimal Working Example
(in the mwe there are no side-effects as long as `lemma` is present, but eg the `draft`-option isn't working)
```` latex
\documentclass[a4paper]{book}

\usepackage[series={A},noeledsec,noledgroup]{reledmac}
\usepackage{reledpar}

\begin{document}

\beginnumbering
\pstart
Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi commodo, ipsum sed pharetra gravida, orci magna rhoncus neque, id pulvinar odio lorem non turpis. Nullam sit amet enim. Suspendisse id velit vitae ligula volutpat condimentum. Aliquam erat volutpat. Sed quis velit. Nulla facilisi. Nulla libero. Vivamus pharetra posuere sapien. Nam consectetuer. Sed aliquam, nunc eget euismod ullamcorper, lectus nunc ullamcorper orci, fermentum bibendum enim nibh eget ipsum. Donec porttitor ligula eu dolor. Maecenas vitae nulla consequat libero cursus venenatis.
Duis eros mi, dictum vel, %
\edtext{%
\edtext{fringilla sit}{%
	\Afootnote{}}
 %
\edtext{amet}{%
	\Afootnote{}}%
, fermentum id, sem.
\pend
\pstart
Fusce %
\edtext{mattis}{%
	\Afootnote{}}
}{%
%%with the lemma there is no error message:
	\lemma{fringilla–mattis}
	\Afootnote{del.}}
 est id diam.
\pend

\endnumbering

\end{document}
````

**Comment by maieul** [comment]
That was just an hasardious thing that you get an error message when you had a \pstart insided an   \edtext.

Please try branch issue763, which add real error message for theses cases.

**Comment by floriandk** [comment]
Working perfectly well. 
That is a great help in tidying up my code: I have already found other two occurrences in my project; manually it would have taken me days. This is a sanity saving addition -- highly appreciated!

(The code in question is converted from a source file that had no structural restrictions, which explains that there are quite some problems of this type)

It just occured to me that it might be a good idea to attach the same warning also to occasions where `\edtext` spans `&` or `\&` in stanzas. Afaics this always gives `Extra }, or forgotten \endgroup.` but the new warning seems far clearer to me.

**Comment by maieul** [comment]
please open new issue. I close this was, I do the release right now
